### DISCUSSION TITLE
Task Assignment - Wiki

### DISCUSSION THREAD

**Discussion by sujanajayram** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#49] [General]
Hi everyone,
The below are the wiki pages list. please respond with your name, task no and task name.

1 Introduction
2 Docker
3 Jupyter
4 Mamba
5 Streamlit
6 Scout
7 Mistral/Milvus
8 LangChain/FAISS
9 Nemo Curator/Guardrails
10 SRS
11 10 github features
12 System design (low-fidelity) & architecture diagrams
13 DoS
14 BSCE
15 ZAP

**Comment by sujanajayram** [comment]
I will proceed with the below
Name: sujana
Task: 12 System design (low-fidelity) & architecture diagrams

**Comment by farheen-akhter-code** [comment]
Task 7 - Farheen Akhter

**Comment by GorrepatiAbhishek** [comment]
I want to go with docker
Name: Abhishek
Task: 2

**Comment by ERIGELASANDHYA** [comment]
Name: Sandhya Erigela
Task: 13- DoS

**Comment by JasonHernandez24** [comment]
Name: Jason Hernandez
Task: 3 Jupyter

**Comment by VishnuTejaG07** [comment]
Name: Gurram Vishnu Teja
Task:  8 LangChain/FAISS

**Comment by ketha-shishir** [comment]
Name: Ketha Shishir
Task: 4 Mamba

**Comment by CSUSB-Jesse-Hernandez** [comment]
Name: Jesse Hernandez
Task 5: Streamlit

**Comment by Siyamul33** [comment]
Name: Siyamul

Task 14 bsce

**Comment by Arungodugu** [comment]
Name: Arun
Task 11: 10 github features

**Comment by shreya192-edu** [comment]
Name : Shreya
Task - 6 Scout

**Comment by Prasanth-Kandikatla** [comment]
Name: Prasanth 
Task: 9 Nemo Curator/Guardrails

**Comment by Jahnavi24csusb** [comment]
Name: Jahnavi
10

**Comment by VedakshariKanneti** [comment]
Name: Vedakshari 
Task:15
